playbooks/openshift-logging/private/config.yml

```diff
@@ -16,7 +16,7 @@
 # Normally we only collect this information for our master group entries
 # we want to also collect this for nodes so we can match group entries to nodes
 - name: Get common IP facts when necessary
-  hosts: all
+  hosts: all:!oo_masters
```

Comment:
We don't want to operate against all group.  Let's update this to
oo_nodes_to_config:oo_etcd_to_config:!oo_masters

Reply:
I'm not sure we need to even include `oo_etcd_to_config` in this case, do we? we should only be pulling this new fact for the nodes since we use `oo_masters_to_config` initially